### ISSUE TITLE
Repository Links

### ISSUE THREAD

**Issue by MAlexGR** [google/docsy-example#98]
Hi there!

According to user guide, in order to [configure links](https://www.docsy.dev/docs/adding-content/repository-links/#link-configuration), which directs reader to my repo ("Edit this Page", "Create child page" etc), I have to configure my `config.yaml` by adding the line: `github_repo = "https://github.com/<my_user_name>/<my_repo>"` (am I right or I miss something?). I dit that but now the links includes the google/docsy-example's addresses along with my project's folders, something like `https://github.com/google/docsy-example/edit/master/content/gr/docs/_index.md` (I suppose the `https://github.com/google/docsy-example/edit/master/` part is a link to original "google/docsy-example" repo and the `content/gr/docs/_index.md`, a link to my repo). 

Why, after declaring my repo in config.yaml, links are still referencing to docsy-example? 

This issue is similar to #96 (also mine), where I'm describing that, at the very bottom of every page, a message like: "Last modified 16.12.2020: first commit (f91b5bc)" appears, where the `first commit (f91b5bc)` is linkable. This link directs to docsy-example repo (wich ends in a 404 page error) and not to my GitHub repo.

In short, it seems that from the first moment I installed and configured docsy in my project, it continues to be "connected" to `google/docsy-example`. I know that in your [instructions](https://www.docsy.dev/docs/getting-started/#using-the-command-line) you clarify this and I followed them faithfully but the above problems continue to exist.

Thank you!
Alex

**Comment by MAlexGR** [comment]
@LisaFC First of all, Happy New Year to everybody!

Sorry for late response but I changed my laptop's SSD drive and I had to rebuild everything from scratch, including the local copy of my site repo (clone from my remote).

During this proccess I found that Hugo could't serve or build my site. Searching the issue I found that docsy theme wasn't present in `themes/` folder. To make things short, I found that I had to update docsy submodule (I added docsy as submodule when I created my site).

Why I'm writing this? Your documentation is really comprehensive but not so clear for someone like me (medium or no experience with Hugo, HTML, SCSS, etc and git  internals). So when someone has to work on the same project from start (for example from another machine), he/she has to update all submodules  manually. This actually means to repeat the `git submodule update --init --recursive` (["Using Docsy as submodule"](https://www.docsy.dev/docs/getting-started/#using-the-docsy-theme-as-a-submodule)). This may be trivial information, but it's my experience. Use it if you want.

As for the original question, Yes you are right I forgot to change the `github_repo` param. Thank you!

Alex
